frontend/src/hooks/useDashboardFilterKey.js

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'react';
+
+const FILTER_KEY = (dashboardName, reportType) => `${dashboardName}-filters-${reportType}`;
+
+export default function useDashboardFilterKey(
+  dashboardName,
+  reportType = '',
+) {
+  const filterKey = useMemo(() => FILTER_KEY(
```

Comment:
Do we need a hook/memo for this? I guess it might not matter.

https://blog.logrocket.com/when-not-to-use-usememo-react-hook/

Reply:
I memoized this to stop multiple data fetches